<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1495</width>
    <height>1265</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>SUPER MASS SPECTREMULATOR</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>360</y>
      <width>651</width>
      <height>401</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>10</number>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>200</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>35</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>35</number>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>Faraday A</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Faraday B</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ion Counting</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Initialise Mass Spec Optics</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <number>6000</number>
    </property>
    <property name="singleStep">
     <number>50</number>
    </property>
    <property name="value">
     <number>600</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>190</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <number>50000</number>
    </property>
    <property name="singleStep">
     <number>200</number>
    </property>
    <property name="value">
     <number>2700</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Resolution</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Mass Resolving Power</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Collector Configuration</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_res">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>140</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_MRP">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>190</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_start_em">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>40</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>START EMULATION</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_connect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stop_em">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>100</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_comports">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>COMPort</string>
    </property>
   </widget>
   <widget class="QTableView" name="CollectorTableView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>431</width>
      <height>401</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::NoPen</enum>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>25</number>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_com">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>310</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mass: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_mass">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>320</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_timer">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>320</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>310</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Integration Time:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="Peaks_tableWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>770</y>
      <width>1091</width>
      <height>401</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="rowCount">
     <number>10</number>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <property name="align" stdset="0">
     <string/>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>265</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>35</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>35</number>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>Mass</string>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ion beam signal</string>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Collector channel</string>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Detector Type</string>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="0" column="3">
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="1" column="3">
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="2" column="3">
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="3" column="3">
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="4" column="2">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="4" column="3">
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="5" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="5" column="2">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="5" column="3">
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="6" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="6" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="6" column="2">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="6" column="3">
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="7" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="7" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="7" column="2">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="7" column="3">
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="8" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="8" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="8" column="2">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="8" column="3">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="9" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="9" column="1">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="9" column="2">
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
    <item row="9" column="3">
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_peak_gen">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>770</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Read Peaks</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1495</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
